kubernetes/build.gradle

```diff
@@ -41,14 +53,25 @@ task createRDockerfile(type: Dockerfile) {
 task createPythonDockerfile(type: Dockerfile) {
   destFile = outputFilePython
   arg "spark_version"
-  from 'spark-py:$spark_version'
+  if (VersionNumber.parse(sparkVersion) >= VersionNumber.parse("2.4.0")) {
+    from 'spark-py:$spark_version'
+  } else {
+    from 'spark:$spark_version'
+  }
+  if (VersionNumber.parse(sparkVersion) < VersionNumber.parse("2.4.0")) {
+    runCommand "apk add --update py-pip"
+  }
   copyFile("h2o_pysparkling_${sparkMajorVersion}-${version}.zip", "/opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip")
-  runCommand """\\
-                pip install /opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip && \\
+  runCommand "pip install /opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip"
+  if (VersionNumber.parse(sparkVersion) >= VersionNumber.parse("2.4.4")) {
```

Comment:
Shouldn't be there `2.4.5` on this line?

Reply:
Yes, good catch!